pkg/controller/linstorsatelliteset/linstorsatelliteset_controller.go

```diff
@@ -1148,9 +1149,9 @@ func newMonitoringConfigMap(set *piraeusv1.LinstorSatelliteSet) *corev1.ConfigMa
 		Data: map[string]string{
 			"prometheus.toml": fmt.Sprintf(`
 [[prometheus]]
-address = "0.0.0.0:%d"
+address = "%s:%d"
 enums = true
-`, monitoringPort),
+`, set.Spec.MonitoringBindAddress, monitoringPort),
```

Comment:
I think we need to handle the "empty" case here, since this is a new field that might not be set. Configuring drbd-reactor with `address = ":9942"` does not work. interestingly, it doesn't produce any errors, but it also is not reachable from anywhere.

Reply:
Thanks, fixed